crates/bevy_app/src/plugin.rs

```diff
@@ -25,8 +25,8 @@ pub trait Plugin: Downcast + Any + Send + Sync {
     /// Configures the [`App`] to which this plugin is added.
     fn build(&self, app: &mut App);
 
-    /// Has the plugin finished it's setup? This can be useful for plugins that needs something
-    /// asynchronous to happen before they can finish their setup, like renderer initialization.
+    /// This will be called when the plugin has finished its setup. Useful for plugins that need something
```

Comment:
this is not right, this is called to check if the plugin has finished its setup

Reply:
Ok, I’ll just revert the changes